src/handlers/test-run-handler.ts

```diff
@@ -18,10 +18,12 @@ export class TestRunHandler {
     private readonly protocolHandler: MutationServer,
     private readonly testControllerHandler: TestControllerHandler,
   ) {
-    this.testController.createRunProfile('Mutation', vscode.TestRunProfileKind.Run, (request) => this.mutationRunHandler(request));
+    this.testController.createRunProfile('Test mutations', vscode.TestRunProfileKind.Run, (request, token) =>
```

Comment:
```suggestion
    this.testController.createRunProfile('Test mutations', vscode.TestRunProfileKind.Run, this.mutationRunHandler.bind(this)
```

dit kan hier volgens mij ook